SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -260,7 +260,7 @@ const FString DEVELOPMENT_AUTH_PLAYER_ID = TEXT("Player Id");
 const FString SCHEMA_DATABASE_FILE_PATH  = TEXT("Spatial/SchemaDatabase");
 const FString SCHEMA_DATABASE_ASSET_PATH = TEXT("/Game/Spatial/SchemaDatabase");
 
-const FString ZoningAttribute = DefaultServerWorkerType.ToString();
+const WorkerAttributeSet LoadBalancerAttributeSet = { DefaultServerWorkerType.ToString() };
```

Comment:
I believe this should be set to SpatialGDKSettings::LoadBalancingWorkerType

Reply:
this would be a logical change as part of this PR, and also is tricky because I'm not sure if it's a good idea to access the settings in spatial constants- do you think all the accessors should be changed to use the settings instead? And have the settings return the attribute set?

Reply:
You could make a function that attempts to grab the value from the settings, and fallbacks to the DefaultServerWorkerType if it fails.

Reply:
In the same way as the `GetClientAuthorityComponent` function?